website/src/pages/api/chat/index.ts

```diff
@@ -13,7 +15,16 @@ const handler = withoutRole("banned", async (req, res, token) => {
     if (req.query.chat_id) {
       data = await client.get_chat(req.query.chat_id as string);
     } else {
-      data = await client.get_my_chats();
+      const params = {
```

Comment:
please add type `params: GetChatParams` just to be sure.

Otherwise looks good!

Reply:
Oh I miss type this, added